full_install.sh

```diff
@@ -0,0 +1,26 @@
+deactivate
+rm -rf ./venv
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip install -U pip
```

Comment:
--upgrade

Reply:
+ We probably don't want this here, it should be in instructions that user should have already upgraded pip prior to calling this.

Reply:
-U is the same as --upgrade